src/fastertransformer/cuda/Dispatch.h

```diff
@@ -107,21 +107,14 @@ CastedTuple castArgs(const std::tuple<Args...>& args) {
     std::apply(func_name<T>, typed_args);                                                   \
 }
 
-#if USING_CUDA
+#if USING_CUDA || USING_ROCM
```

Comment:
this macro condition should be removed since it's used under cuda or rocm.

Reply:
Thanks, removed it.